### PR TITLE
imager/README.txt: typo, grammar, style fixes

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: imager/README.txt [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
+The easiest method for creating EEPROM rescue images, and formatting SD 
+cards, is to use Raspberry Pi Imager from https://raspberrypi.com/software.
+Raspberry Pi Imager provides a GUI for downloading the latest version of 
+this rescue image and flashing it to an SD card.
 
 Alternatively, copy the contents of this zip file to a blank
-FAT formatted SD-CARD. The FAT partition must be < 32 GB.
+FAT formatted SD card. The FAT partition must be < 32 GB.
 
-To update the EEPROMs:
```
Comment (timg236) [inline imager/README.txt:19]: There's more than one EEPROM on the intialise Pi4B

Comment (andrum99) [reply imager/README.txt:19]: I know. The page does not explain this thought - it mentions the utility will update the VL805 firmware, but it does not explain that this lives on a separate EEPROM chip on Pi 4B v1.1 and 1.2. I will add an explanation of this.

File: imager/README.txt [hunk 2]
```diff
@@ -1,33 +1,34 @@
...
+If successful, the green LED on the Raspberry Pi will blink rapidly forever.
+An unsuccessful update of the EEPROM is indicated by a different blinking 
+pattern corresponding to the specific error.
 
-If a HDMI display is attached then the screen will display green for success
-or red if a failure occurs.
+If an HDMI display is attached, then the screen will display green for
+success or red if a failure occurs.
 
-The SD-CARD can now be removed and re-formatted using the Raspberry Pi Imager
```
[the remark this answers is not in the document]
Comment (andrum99) [reply imager/README.txt:32]: I hadn't realised this was the case, so I think that probably needs explaining.

File: imager/README.txt (hunk 1 above)
Comment (andrum99) [reply imager/README.txt:19]: Fixed.

File: imager/README.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (andrum99) [reply imager/README.txt:32]: Fixed.

File: imager/README.txt [hunk 3]
```diff
@@ -1,33 +1,42 @@
...
-This rescue image also updates the USB 3.0 (VL805) firmware to the latest
-version (138a1) with better full-speed Isochronous endpoint support.
+This rescue image also updates the USB 3 controller (VL805) firmware to the
+latest version, 138a1, which has better full-speed isochronous endpoint 
+support.
 
-This easiest method for creating EEPROM rescue images or formatting sdcards
-is to use the Raspberry Pi Imager from https://raspberrypi.com/software.
-The imager provides a GUI for downloading the latest version of this rescue
-image and flashing it to a spare SD CARD.
```
Comment (timg236) [inline imager/README.txt:14]: Restore “spare” because this will deltas whatever was on the card. So we always use the word spare to imply that

File: imager/README.txt [hunk 4]
```diff
@@ -1,33 +1,42 @@
...
-FAT formatted SD-CARD. The FAT partition must be < 32 GB.
+FAT formatted SD card. The FAT partition must be < 32 GB.
 
-To update the EEPROMs:
+To update the EEPROM:
 
 1. Power off the Raspberry Pi
-2. Insert the SD-CARD
-3. Power on Raspberry Pi
+2. Insert an SD card containing the files from this zip file
```
Comment (timg236) [inline imager/README.txt:28]: A bit wordy. Perhaps “bootloader update SD card”

Comment (andrum99) [reply imager/README.txt]: I prefer my wording, but I've change it to yours.

File: imager/README.txt [hunk 5]
```diff
@@ -1,33 +1,42 @@
...
 
-The Raspberry Pi 4 has a small EEPROM used to store the bootloader.
+The Raspberry Pi 4 contains a small EEPROM used to store the bootloader.
 
 This rescue image reverts the bootloader EEPROM to factory default settings.
 
-This rescue image also updates the USB 3.0 (VL805) firmware to the latest
-version (138a1) with better full-speed Isochronous endpoint support.
+This rescue image also updates the USB 3 controller (VL805) firmware to the
+latest version, 138a1, which has better full-speed isochronous endpoint
```
Comment (lurch) [inline imager/README.txt:9]: @timg236 I wonder if this should just say "latest version" and omit the "138a1" ? :shrug:
